### PR TITLE
AMSSoftware.BusinessCardMaker version 1.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93835)

### CODE REVIEW

**Review by OverRevvv — DISMISSED** [review]

File: manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AMSSoftware.BusinessCardMaker
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: AMS Software
+PackageName: Business Card Maker
+License: Freeware
```
Comment (stephengillie) [inline manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.locale.en-US.yaml:9]:
```suggestion
PublisherUrl: https://ams-photo-software.com
PublisherSupportUrl: https://ams-photo-software.com/support.php
PrivacyUrl:   https://ams-photo-software.com/privacy-policy.php
PackageName: Business Card Maker
PackageUrl: https://ams-photo-software.com/business-card-maker/
License: Freeware
LicenseUrl: https://ams-photo-software.com/license-agreement.php
```

File: manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AMSSoftware.BusinessCardMaker
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.installer.yaml:5]:
```suggestion
PackageVersion: 9.15
```
Should Match Control Panel

File: manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AMSSoftware.BusinessCardMaker
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 9.15
```
Should Match Control Panel

File: manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: AMSSoftware.BusinessCardMaker
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.yaml:5]:
```suggestion
PackageVersion: 9.15
```
Should Match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/93835/files#r1071702131

**Review by Trenly — COMMENTED** [review]
Folder needs to be renamed to match the version

**Review by stephengillie — APPROVED** [review]

**Pull request merged**